components/LoginModal.vue

```diff
@@ -52,7 +47,7 @@
           data-testid="forgotten-password-button"
           @click="setIsForgottenValue(true)"
         >
-          Forgotten password?
+          Forgot Password?
```

Comment:
use i18n lang helper instead of hardcoding the text

Reply:
Resolved